Fortnox.NET/Models/Invoice/InvoiceRow.cs

```diff
@@ -1,40 +1,120 @@
 using FortnoxNET.Utils;
+using Newtonsoft.Json;
 
 namespace FortnoxNET.Models.Invoice
 {
+    [JsonConverter(typeof(CustomJsonConverter))]
     public class InvoiceRow
     {
+        /// <summary>
+        /// Account number. If not provided the predefined account will be used.
+        /// </summary>
         public int AccountNumber { get; set; }
+
+        /// <summary>
+        /// Article number
+        /// </summary>
         public string ArticleNumber { get; set; }
+
+        /// <summary>
+        /// Contribution Percent
+        /// </summary>
         [JsonReadOnly]
-        public string ContributionPercent { get; set; }
+        public decimal ContributionPercent { get; set; }
+
+        /// <summary>
+        /// Contribution Value
+        /// </summary>
         [JsonReadOnly]
-        public string ContributionValue { get; set; }
+        public decimal ContributionValue { get; set; }
+
+        /// <summary>
+        /// Code of the cost center for the row. The code must be of an existing cost center
+        /// </summary>
         public string CostCenter { get; set; }
-        public string DeliveredQuantity { get; set; }
+
+        /// <summary>
+        /// Delivered quantity
+        /// </summary>
+        public decimal DeliveredQuantity { get; set; }
+
+        /// <summary>
+        /// Row description
+        /// </summary>
         public string Description { get; set; }
+
+        /// <summary>
+        /// Discount amount
+        /// </summary>
         public decimal Discount { get; set; }
+
+        /// <summary>
+        /// The type of discount used for the row. Can be AMOUNT or PERCENT
+        /// </summary>
         public string DiscountType { get; set; }
+
+        /// <summary>
+        /// If the row is housework
+        /// </summary>
         public bool HouseWork { get; set; }
+
+        /// <summary>
+        /// Hours to be reported if the quantity of the row should not be used as hours.
+        /// Can only contain numeric values without decimals
+        /// </summary>
         public int? HouseWorkHoursToReport { get; set; }
+
+        /// <summary>
+        /// The type of house work. Can be
+        /// CONSTRUCTION ELECTRICITY GLASSMETALWORK
+        /// GROUNDDRAINAGEWORK MASONRY
+        /// PAINTINGWALLPAPERING HVAC MAJORAPPLIANCEREPAIR
+        /// MOVINGSERVICES ITSERVICES CLEANING TEXTILECLOTHING SNOWPLOWING
+        /// GARDENING BABYSITTING OTHERCARE OTHERCOSTS or empty.
+        /// </summary>
         public string HouseWorkType { get; set; }
```

Comment:
Can we represent these as an enum and resolve it via name lookup in the JSON converter? (out of scope for this PR, but can be done separately CC @brokenprogrammer)

Reply:
This should be possible yes and something that we do want to add in the future.